cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflow.java

```diff
@@ -106,4 +106,12 @@ protected final void addMapReduce(String mapReduce) {
   protected final void addSpark(String spark) {
     configurer.addSpark(spark);
   }
+
+  /**
+   * Adds a fork to the {@link Workflow}
```

Comment:
Looks like we turned off the checkstyle for that. But it's recommended (http://checkstyle.sourceforge.net/config_javadoc.html#JavadocStyle and http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#firstsentence)